cdc/processor/sinkmanager/table_sink_wrapper.go

```diff
@@ -125,7 +128,12 @@ func (t *tableSinkWrapper) updateResolvedTs(ts model.ResolvedTs) error {
 }
 
 func (t *tableSinkWrapper) getCheckpointTs() model.ResolvedTs {
-	return t.tableSink.GetCheckpointTs()
+	currentCheckpointTs := t.checkpointTs.Load()
+	newCheckpointTs := t.tableSink.GetCheckpointTs()
+	if currentCheckpointTs > newCheckpointTs.ResolvedMark() {
```

Comment:
In which scenario will this case happen?
- `newCheckpointTs.ResolvedMark()` is zero
- Is there any other scenarios?

Reply:
Only when the table is just added.